src/depackers/miniz_zip.h

```diff
@@ -0,0 +1,246 @@
+
+#pragma once
+#include "miniz.h"
```

Comment:
This applies to miniz.h too, but include-guards should be more portable than `#pragma once`, right?

Reply:
Nice, will change shortly.

What is the thing with the CI macos failure???

Reply:
It's a clang crash bug that (I thought) happens infrequently with parallel builds. It also happened in one of my AliceLR/libxmp regression test runs, but so far it doesn't seem to happen with the other clang-based regression tests.

I see it sometimes when building with clang + sanitizers in Fedora, so it might specifically be a thing for newer clang versions. It *only* ever happens with test-dev (not the main executable) in my experience.

Reply:
Replaced those `#pragma once` with header guards.

Reply:
> It's a clang crash bug that (I thought) happens infrequently with parallel builds. It also happened in one of my AliceLR/libxmp regression test runs, but so far it doesn't seem to happen with the other clang-based regression tests.
> 
> I see it sometimes when building with clang + sanitizers in Fedora, so it might specifically be a thing for newer clang versions. It _only_ ever happens with test-dev (not the main executable) in my experience.
> 
> edit: I don't know _why_ it would only happen with parallel builds since that should only matter for Make, not clang.

Never happened here before, and now it didn't happen after my header guard commit.
Does apple and/or llvm guys know about it, I wonder..

Reply:
Yeah, it only happens *sometimes*. I don't know what causes it, but this gets spammed a bunch, so maybe it's a hint?
```
warning: null character ignored [-Wnull-character]
```

See #526 is this keeps being a problem going forward.